### PR TITLE
Updated Apple's Jobs link.

### PR DESCRIPTION
**Pull request by SaadAnsari17**
The previous link was not working.

### CODE REVIEW

**Review by SaptarshiSarkar12 — APPROVED** [review]
The new link is working perfectly as of now, and the changes need to be merged with the main branch as soon as possible.

**Comment by priynshuchouhn** [comment]
But this job link is not for open source software engineers. I guess

**Review by priynshuchouhn — CHANGES_REQUESTED** [review]
But this job link is not for open source software engineers. I guess

File: Open-Source/Job-Boards/Readme.md [hunk 1]
```diff
@@ -7,4 +7,4 @@
 - [Open Source Engineer](https://www.indeed.com/q-Open-Source-Engineer-jobs.html?vjk=1475599e24cc62aa) - Indeed
 - [Open Source Engineer](https://www.aquasec.com/about-us/careers/co/engineering/81.E1B/open-source-engineer/all/) - Aquasec
 - [RedHat Careers](https://www.redhat.com/en/jobs)
-- [Software Engineer - Open Source](https://jobs.apple.com/en-us/details/200385788/software-engineer-open-source) - Apple
+- [Software Engineer - Open Source](https://jobs.apple.com/en-us/search?search=Software%20Engineer&sort=relevance&location=united-states-USA) - Apple
```
Comment (kaiwalyakoparkar) [inline Open-Source/Job-Boards/Readme.md:10]: please make the location settings to worldwide and not just united states

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Brittle

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Comment by kaiwalyakoparkar** [comment]
Also please resolve this merge conflict

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

File: Open-Source/Job-Boards/Readme.md (hunk 1 above)
Comment (SaadAnsari17) [reply Open-Source/Job-Boards/Readme.md]: Done! +resovled the merge conflict.

**Comment by priynshuchouhn** [comment]
#57 I already Removed this link because the relevant job is not available.

**Comment by kaiwalyakoparkar** [comment]
@priynshuchouhn you are correct. Please make sure you mention what you are planning to contribute clearly through your issue this will reduce the iterative work for both the end.

@SaadAnsari17 so sorry but I will have to close this pr and issue as this is already being taken care of. Looking forward to your contribution on other issues :)

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Pull request closed without merging**